dell/anomaly_alert.py

```diff
@@ -0,0 +1,117 @@
+import requests
+import time
+import json
+import pendulum
+import pickle
+import logging
+import configparser
+
+
+logging.basicConfig(level=logging.WARNING)
+
+
+def log_in(host, user_name, password, user_agent):
+    TOKEN = "token"
+    USER_AGENT = "User-Agent"
+    X_CSRF_TOKEN = "X-CSRF-Token"
+    url = host + '/api/v1/login-check'
+    headers = {"User-Agent": user_agent}
+    data = {"userName":user_name, "password":password}
+    resp = requests.post(url, data=data, headers=headers, verify=False)
+    assert resp.status_code == 200, "failed to login!"
+    logging.debug("Successfully login and get the token")
+    headers = {USER_AGENT: user_agent, X_CSRF_TOKEN: json.loads(resp.text)[TOKEN]}
+    return resp.cookies, headers
+
+
+def get_anomaly_data(host, headers, cookies, data):
+    url = host + '/api/v2/projectanomalytransfer'
+    resp = requests.get(url, params=data, headers=headers, cookies=cookies, verify=False)
+    logging.debug(resp.status_code)
+    assert resp.status_code == 200, "failed to get anomaly data!"
+    return json.loads(resp.text)
+
+
+def get_anomaly_events(start, end, events_sent, host, user_name, password):
+    startTime = int(start.timestamp()*1000)
+    endTime = int(end.timestamp()*1000)
+
+    data = {"projectName": "TD_metric@demoUser", "transferToProjectName": "dummy", "transferToCustomerName": "user",
+            "startTime": startTime, "endTime": endTime}
+    user_agent = "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.122 Safari/537.36"
+
+    (cookies, headers) = log_in(host, user_name, password, user_agent)
```

Comment:
we don't need the header anymore, but we need the cookie to have the project's userName